examples/plot_linestrengths.py

```diff
@@ -49,19 +57,15 @@ def _calc_Q(molecule, iso, T_ref, T_gas):
 
     iso_arr = list(range(max(iso_set) + 1))
 
-    Qref_arr = np.empty_like(iso_arr, dtype=np.float64)
-    Qgas_arr = np.empty_like(iso_arr, dtype=np.float64)
+    Qref_Qgas_ratio = {}
+
     for iso in iso_arr:
         if iso in iso_set:
```

Comment:
@gagan-aryan one more comment : why do you need to create iso_arr ? Can't we just do `for iso in iso_set` ?